docs/data/whats-new/WhatsNewLayout.js

```diff
@@ -0,0 +1,153 @@
+import * as React from 'react';
+import Box from '@mui/material/Box';
+import Link from '@mui/material/Link';
+import List from '@mui/material/List';
+import ListItem from '@mui/material/ListItem';
+import Card from '@mui/material/Card';
+import CardActions from '@mui/material/CardActions';
+import CardContent from '@mui/material/CardContent';
+import Button from '@mui/material/Button';
+import Typography from '@mui/material/Typography';
+
+function BlogCard(props) {
+  return (
+    <Card
+      sx={{
+        margin: '10px',
+        flexBasis: '350px',
+        display: 'flex',
+        flexFlow: 'column',
+        boxShadow: '0px 1px 3px 0px rgba(0,0,0,0.25)',
+      }}
+    >
+      <CardContent sx={{ p: 0 }}>
+        <Box sx={{ borderBottom: '1px solid #eee', marginBottom: 1, px: 1 }}>
+          <Typography variant="caption" color="text.secondary">
+            {props.blog.announcementDate}
+          </Typography>
+          <Typography gutterBottom variant="h6" component="div">
+            {props.blog.title}
+          </Typography>
+        </Box>
+
+        <Box sx={{ p: 1 }}>
+          <Typography variant="body2" component="div" color="text.secondary">
+            {props.blog.description}
+          </Typography>
+
+          <List sx={{ listStyle: 'circle', pb: 0 }}>
+            {props.blog.highlightList.map((item) => (
+              <ListItem
+                sx={{
+                  py: 0.5,
+                  px: 0.5,
+                  [`&:before`]: {
+                    content: '"•"',
+                    color: 'grey',
+                  },
+                }}
+              >
+                <Typography sx={{ pl: 1 }} variant="body2">
+                  <Link sx={{ cursor: 'pointer' /*, color: '#818181'*/ }}>
+                    {item}
+                  </Link>
+                </Typography>
+              </ListItem>
+            ))}
+          </List>
+        </Box>
+      </CardContent>
+      <CardActions
+        sx={{ justifyContent: 'center', flex: '1 1 auto', alignItems: 'end' }}
+      >
+        <Button sx={{ width: '100%' }} href={props.blog.url}>
+          Read more
+        </Button>
+      </CardActions>
+    </Card>
+  );
+}
+
+export default function PickersPlaygroundWrapper() {
+  const blogs = [
+    {
+      title: 'MUI X v6.10.0',
+      description: 'A summary of all key new features since v6.0.0.',
+      announcementDate: 'Mon, Jul 31, 2023',
+      url: 'https://deploy-preview-37957--material-ui.netlify.app/blog/mui-x-mid-v6-features/',
+      highlightList: [
+        'New UI for TimePickers (Digital Clock)',
+        'Support for timezone on Date and Time Pickers',
+        'Copy and Paste on Data Grid',
+        'Filters on Data Grid column headers',
+        'Charts Alpha 🧪',
+      ],
```

Comment:
If they are supposed to be links.
```suggestion
      highlightList: [
        { title: 'New UI for TimePickers (Digital Clock)', url: 'path/to/doc/for/this/highlight' },
        { 'Support for timezone on Date and Time Pickers', url: 'path/to/doc/for/this/highlight' },
        { 'Copy and Paste on Data Grid', url: 'path/to/doc/for/this/highlight' },
        { 'Filters on Data Grid column headers', url: 'path/to/doc/for/this/highlight' },
        { 'Charts Alpha 🧪', url: 'path/to/doc/for/this/highlight' },
      ],
```